LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2823,6 +2823,45 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                         break;
 
+                    case Model.ROG_MAXIMUS_XI_FORMULA: //NC6798D
+                        v.Add(new Voltage("Vcore", 0));
+                        v.Add(new Voltage("+5V", 1, 4, 1));
+                        v.Add(new Voltage("AVSB", 2, 34, 34));
+                        v.Add(new Voltage("3VCC", 3, 34, 34));
+                        v.Add(new Voltage("+12V", 4, 11, 1));
+                        v.Add(new Voltage("VIN8", 5));
+                        v.Add(new Voltage("CPU Gfx", 6));
```

Comment:
Can you follow naming as done in this file? Some of the names here don't match like this one.

Reply:
TBH I took the names from HWinfo. I tried to align the names as much as possible. HWinfo had a name for "Voltage #6", I used that. DRAM and DIMM are also used for the same, but I renamed mine to DIMM which had the majority count.